docs/components/ColorPicker.md

```diff
@@ -19,32 +19,34 @@ import React from 'react';
 import { View, ColorPicker } from 'magic-script-components';
 
 export default class MyApp extends React.Component {
-    onColorChanged = event => {
-        // event.Color
-        console.log("onColorChanged event received: ", event);
-    };
-
-    onColorConfirmed = event => {
-        // event.Color
-        console.log("onColorConfirmed event received: ", event);
-    };
-
-    onColorCanceled = event => {
-        // event.Color
-        console.log("onColorCanceled event received: ", event);
-    };
-
-    render() {
-        return (
-            <View name="main-view">
-                <ColorPicker
-                    color={[1, 0.5, 1, 0.8]}
-                    onColorChanged={this.onColorChanged}
-                    onColorConfirmed={this.onColorConfirmed}
-                    onColorCanceled={this.onColorCanceled} />
-            </View>
-        );
-    }
+  onColorChanged = event => {
+    // event.Color
+    console.log('onColorChanged event received: ', event);
+  };
+
+  onColorConfirmed = event => {
+    // event.Color
+    console.log('onColorConfirmed event received: ', event);
+  };
+
+  onColorCanceled = event => {
+    // event.Color
+    console.log('onColorCanceled event received: ', event);
+  };
+
+  render() {
+    return (
+      <View name='main-view'>
+        <ColorPicker
+          height={0.15}
+          color={[1, 0.5, 1, 0.8]}
+          onColorChanged={this.onColorChanged}
+          onColorConfirmed={this.onColorConfirmed}
+          onColorCanceled={this.onColorCanceled} />
+        />
```

Comment:
This looks like a mistake.  You already have a closing `/>` on the line right above.

Reply:
thanks, good catch :+1: